joomla/joomla-cms
# Disable inline JavaScript when directly open SVG files

### Summary of Changes

Disable inline JavaScript when directly open SVG files.
This PR adds an SVG file only CSP rule to protect against JavaScript code embedded in SVG files.

This issue was inital reported to the JSST by Lee Thao

### Testing Instructions

upload an SVG file with this content to the images folder:
```svg
<?xml version="1.0" encoding="UTF-8"?>
<svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
  <circle cx="15" cy="15" r="15" onclick="alert('svg inline script executed!')"></circle>
</svg>
```
try to access that image directly in the browser and click on the black circle.
Please notice the message "svg inline script executed!"
Apply the changes in this PR to the htaccess file.
Reload & click the circle again.
There is no message any more.

### Actual result BEFORE applying this Pull Request

When accessing SVGs from outside of image tags JavaScript can be executed that could lead to XSS

### Expected result AFTER applying this Pull Request

With the proposed changes here we apply a dedicated CSP rule to SVG files that block all inline JS.

### Documentation Changes Required

none

cc @HLeithner @SniperSister 

### Postinstall message

![image](https://user-images.githubusercontent.com/2596554/89122250-0cb4a580-d4c6-11ea-91a2-5a5ed64b9f0d.png)
